README.md

```diff
@@ -24,7 +24,7 @@ This client library supports all of Stytch's live products:
   - [x] [WhatsApp passcodes](https://stytch.com/docs/api/whatsapp-send)
   - [x] [Email passcodes](https://stytch.com/docs/api/send-otp-by-email)
   - [x] [Session Management (Beta)](https://stytch.com/docs/api/sessions-overview)
-  - [ ] [WebAuthn (Coming soon)](https://stytch.com/docs/api/webauthn-overview)
+  - [ ] [WebAuthn (Beta)](https://stytch.com/docs/api/webauthn-overview)
```

Comment:
I think this counts! Or is there more needed here to support WebAuthn as we have it now?
```suggestion
  - [x] [WebAuthn (Beta)](https://stytch.com/docs/api/webauthn-overview)
```

Reply:
8148e73